org.eclipse.zest.tests/src/org/eclipse/zest/tests/examples/AbstractGraphTest.java

```diff
@@ -129,7 +129,7 @@ private void doTest(Snippet annotation, Statement statement) throws Throwable {
 				robot = new GraphicalRobot(graph);
 				shell = graph.getShell();
 				// Wait for layout to be applied
-				waitEventLoop(0);
```

Comment:
We currently only wait a single cycle for the layout operation to complete. Calling readAndDispatch() within the animation seems to mess with scheduling, so this is no longer be enough.